java/connector-framework-internal/src/main/java/org/identityconnectors/framework/impl/api/local/ObjectPool.java

```diff
@@ -122,7 +128,7 @@ public void setNew(final boolean n) {
             isNew = n;
         }
 
-        public void setActive(final boolean v) {
+		public void setActive(final boolean v) {

```

Comment:
Please keep up with indentation, thanks.

Reply:
OK. I'll do. BTW, would be nice to document code formatting on project home page. It is not entirely fair to enforce rules that are not specified anywhere.

Reply:
Thanks.